src/config.js

```diff
@@ -606,36 +646,56 @@ class Config {
   // Returns an {Array} of {Object}s with the following keys:
   //  * `scopeDescriptor` The {ScopeDescriptor} with which the value is associated
   //  * `value` The value for the key-path
-  getAll (keyPath, options) {
+  getAll (keyPath, options = {}) {
     let globalValue, result, scope
```

Comment:
these declarations at the top are probably a relic of decaffeination. I'd just inline them as they're initialized below.